Carthage/Checkouts/DBNetworkStack/Tests/DBNetworkStackTests/RetryNetworkserviceTest.swift

```diff
@@ -59,7 +59,7 @@ class RetryNetworkserviceTest: XCTestCase {
         }, onError: { _ in
             XCTFail()
         })
-        networkServiceMock.returnError(error: .unknownError, count: errorCount)
+        networkServiceMock.returnError(with: .unknownError, count: errorCount)
```

Comment:
Now it reads like return "error with error" implicitly. It may be called `return(error: ...)`.

Not sure how to call the success case then:
1. Call it `return(data: ...)`: Does not directly explain, that it is the success case
2. Keep it as it is: Not consistent to `return(error: ...)`